### DISCUSSION TITLE
Hook to get webpack-dev-server url:port

### DISCUSSION THREAD

**Discussion by TechAkayy** [webpack/webpack#15395] [General]
Hello, 

I'm working on a plugin and I wanted to know if there was a way I can get the current **webpack-dev-server's url & port** (not from config, but which port was actually assigned) inside one of the hooks? Thanks in advance!

**Comment by TechAkayy** [comment]
Anybody can help here? Thanks in advance

**Reply by alexander-akait** [reply]
You can't do it because it is dynamically, you can use webpack dev server API https://github.com/webpack/webpack-dev-server/blob/master/lib/Server.js#L391 in configuration and assign it to dev server options and pass to plugin/loader if you need it

**Reply by TechAkayy** [reply]
Thanks bunch, will check it out 🙏🏾
